values.yaml

```diff
@@ -1,8 +1,15 @@
 # override this to use external elasticsearch
 elasticsearch:
-  host: "elasticsearch-service"
-  port: 9200
+  host: "pelias-elasticsearch-coordinating-only"
```

Comment:
original should be kept, this value should be added in an external values file to override this one. the external values file will not be added to this repo